docs/examples/tensorboard_streaming.rst

```diff
@@ -161,3 +161,8 @@ Now you will be able to see the live training metrics of each client from a cent
 
 The full source code for this exercise can be found in
 `examples/advanced/experiment-tracking/tensorboard <https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/experiment-tracking/tensorboard>`_.
+
+Previous Versions of TensorBoard Streaming
+------------------------------------------
+
+   - `tensorboard-streaming for 2.3 <https://github.com/NVIDIA/NVFlare/tree/2.3/examples/advanced/experiment-tracking/tensorboard-streaming>`_
```

Comment:
Do we need this section?

Reply:
This is also for the future, to keep track of the previous versions when there are more